obspy/io/css/tests/test_core.py

```diff
@@ -27,6 +28,7 @@ def setUp(self):
         # directory where the test files are located
         self.path = os.path.join(os.path.dirname(__file__), 'data')
         self.filename = os.path.join(self.path, 'test.wfdisc')
```

Comment:
Since you suffixed `st_result`, why not change these too instead of a generic 2? Though I prefer a prefix in this case.